packages/svelte/src/compiler/phases/3-transform/client/visitors/BindDirective.js

```diff
@@ -245,8 +245,12 @@ export function BindDirective(node, context) {
 		const has_action_directive =
 			parent.type === 'RegularElement' && parent.attributes.find((a) => a.type === 'UseDirective');
 
-		context.state.after_update.push(
-			b.stmt(has_action_directive ? b.call('$.effect', b.thunk(call)) : call)
-		);
+		if (has_action_directive) {
+			context.state.init.push(
+				b.stmt(has_action_directive ? b.call('$.effect', b.thunk(call)) : call)
```

Comment:
no big deal but seemed no longer necessary to check again.

```suggestion
				b.stmt(b.call('$.effect', b.thunk(call)))
```

Reply:
thanks, this was updated